connectors/spark/src/test/java/io/unitycatalog/connectors/spark/SparkIntegrationTest.java

```diff
@@ -92,17 +93,17 @@ public void testDeltaPathTable() throws IOException, ApiException {
         String path1 = new File(dataDir, "test_delta_path1").getCanonicalPath();
         String tableName1 = String.format("delta.`%s`", path1);
         session.sql(String.format("CREATE TABLE %s(i INT) USING delta", tableName1));
-        Assert.assertTrue(
+        assertTrue(
```

Comment:
stuff like this could be expressed in a more fluent way with AssertJ, such as `assertThat(session.sql("SELECT * FROM " + tableName1).collectAsList()).isEmpty()` instead of using `assertTrue()`. The big upside to this is that AssertJ will show the full content of the collection if the assertion ever fails, which is very helpful when debugging test failures.
A few tests have been already converted in https://github.com/unitycatalog/unitycatalog/commit/ca580fa648211166ad40de79971138e1927024e7. Would you be interested in converting existing assertions to AssertJ in a separate PR?

Reply:
Makes a lot of sense. I will open an issue later today and proceed with these changes. Thank you for the feedback!